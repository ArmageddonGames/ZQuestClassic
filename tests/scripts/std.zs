
//Ensure no deprecations in std.zh!
#option WARN_DEPRECATED error
#option NO_ERROR_HALT on

#include "std.zh"
