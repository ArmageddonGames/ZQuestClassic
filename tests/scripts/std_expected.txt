Compiling 'std.zs'
Pass 1: Parsing
Pass 2: Preprocessing
Pass 3: Registration
Pass 4: Analyzing Code
Pass 5: Checking code paths
Pass 6: Generating object code
Pass 7: Assembling
Success!
Compile finished with exit code '0' (success)

ALLOCATEGMEMV D2,4096; void run() Body Start
SETGVARR 0,D2
QUIT; void run() Body End